compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/pager/Pager.kt

```diff
@@ -345,7 +345,10 @@ object PagerDefaults {
         pagerSnapDistance: PagerSnapDistance = PagerSnapDistance.atMost(1),
         lowVelocityAnimationSpec: AnimationSpec<Float> = LowVelocityAnimationSpec,
         highVelocityAnimationSpec: DecayAnimationSpec<Float> = rememberSplineBasedDecay(),
-        snapAnimationSpec: AnimationSpec<Float> = spring(stiffness = Spring.StiffnessMediumLow),
+        snapAnimationSpec: AnimationSpec<Float> = spring(
```

Comment:
It's a change in default parameters of PUBLIC COMMON method. It's considered as a breaking change by `metalava`/Google.